packages/@react-spectrum/numberfield/test/NumberField.test.js

```diff
@@ -744,11 +765,13 @@ describe('NumberField', function () {
     act(() => {textField.blur();});
     expect(textField).toHaveAttribute('value', '25%');
     expect(onChangeSpy).toHaveBeenLastCalledWith(0.25);
+    expect(onBlurSpy).toHaveBeenCalledTimes(1);
```

Comment:
why do so many tests check onBlur? can we just do it in one
or if there is a specific reason for the others, say why?

Reply:
I was reading the code and tried to check other cases where other actions on the elements shouldn't cause blurs. I can trim them down some.

Reply:
gotcha, for ones you deem necessary, would you mind adding a comment in the test for why we're checking?